src/monorepo-workflow-operations.test.ts

```diff
@@ -1,1788 +1,1214 @@
+            today,
+            stdout,
+            stderr,
+            captureChangesInReleaseBranchSpy,
+          } = await setupFollowMonorepoWorkflow({
+            sandbox,
+            doesReleaseSpecFileExist: false,
+            isEditorAvailable: false,
+          });
+
+          await followMonorepoWorkflow({
+            project,
+            tempDirectoryPath: sandbox.directoryPath,
+            firstRemovingExistingReleaseSpecification: false,
+            today,
+            stdout,
+            stderr,
           });
 
-          describe('when the editor command does not complete successfully', () => {
-            it('removes the release spec file', async () => {
-              await withSandbox(async (sandbox) => {
-                const project = buildMockMonorepoProject();
-                const stdout = fs.createWriteStream('/dev/null');
-                const stderr = fs.createWriteStream('/dev/null');
-                mockDependencies({
-                  determineEditor: {
-                    path: '/some/editor',
-                    args: [],
-                  },
-                });
-                jest
-                  .spyOn(
-                    releaseSpecificationModule,
-                    'waitForUserToEditReleaseSpecification',
-                  )
-                  .mockRejectedValue(new Error('oops'));
-                const releaseSpecPath = path.join(
-                  sandbox.directoryPath,
-                  'RELEASE_SPEC',
-                );
-                await fs.promises.writeFile(releaseSpecPath, 'release spec');
-
-                await expect(
-                  followMonorepoWorkflow({
-                    project,
-                    tempDirectoryPath: sandbox.directoryPath,
-                    firstRemovingExistingReleaseSpecification: true,
-                    stdout,
-                    stderr,
-                  }),
-                ).rejects.toThrow(expect.anything());
-
-                await expect(
-                  fs.promises.readFile(releaseSpecPath, 'utf8'),
-                ).rejects.toThrow(/^ENOENT: no such file or directory/u);
-              });
+          expect(captureChangesInReleaseBranchSpy).not.toHaveBeenCalled();
+        });
+      });
+
+      it('prints a message', async () => {
+        await withSandbox(async (sandbox) => {
+          const { project, today, stdout, stderr } =
+            await setupFollowMonorepoWorkflow({
+              sandbox,
+              doesReleaseSpecFileExist: false,
+              isEditorAvailable: false,
             });
+
+          await followMonorepoWorkflow({
+            project,
+            tempDirectoryPath: sandbox.directoryPath,
+            firstRemovingExistingReleaseSpecification: false,
+            today,
+            stdout,
+            stderr,
           });
+
+          expect(stdout.data()[0]).toMatch(
+            /^A template has been generated that specifies this release/u,
```

Comment:
It looks like we lost the test for ensuring this file gets created. Unless I'm missing it

Reply:
Hmm, yes, it looks like tests are missing for what happens if `validateReleaseSpecification`, `planRelease` or `executeReleasePlan` throws (which would cause the file to not get removed, in the case in which it is created). I'll add those.

Reply:
Okay, added in [aca7cce](https://github.com/MetaMask/create-release-branch/pull/21/commits/aca7cce4a026221f9da66c99bf1cfa66710579c5).